### ISSUE TITLE
bug: Ambiguous documentation

### ISSUE THREAD

**Issue by GeorgeSchafer** [asdf-vm/asdf#1366]
### Describe the Bug

I was trying to install asdf and I ran across these [instructions](https://asdf-vm.com/guide/getting-started.html#_1-install-dependencies:~:text=macOS%3A,%23). The way this is written, it is not clear that I should install homebrew OR spack, not homebrew AND spack.

### Steps to Reproduce

1. Go [here](https://asdf-vm.com/guide/getting-started.html#_1-install-dependencies:~:text=macOS%3A,%23)
2. Install Homebrew
3. Waste two hours trying to install spack needlessly. 🤣 

Note: I am more amused than annoyed, just wanted to give a heads up.

### Expected Behaviour

Instructions explaining that I should install homebrew OR spack

### Actual Behaviour

Instructions implying I need homebrew AND spack

### Environment

```shell
OS:
Darwin [USER] 21.6.0 Darwin Kernel Version 21.6.0: Mon Aug 22 20:17:10 PDT 2022; root:xnu-8020.140.49~2/RELEASE_X86_64 x86_64

SHELL:
zsh 5.8.1 (x86_64-apple-darwin21.0)

ASDF VERSION:
v0.10.2

ASDF ENVIRONMENT VARIABLES:


No plugins installed
ASDF INSTALLED PLUGINS:
```

**Comment by Stratus3D** [comment]
The instructions don't say either one should be installed. The assumption was you'd run the appropriate command for your package manager (the odd one being Brew which doesn't need any dependencies installed in advance).

Can you provide a link to docs for a similar project that don't have this issue? Most projects end up displaying install/setup instructions in a matrix like we do because there are different OS/package manager combinations.

**Comment by jthegedus** [comment]
I have a suggested improvement in a comment in #1388 to address this, though spelling out this specific scenario is a bit much IMO.

**Comment by jthegedus** [comment]
Closed in #1388
